controllers/yawol-cloud-controller/targetcontroller/node_controller_test.go

```diff
@@ -148,6 +148,27 @@ var _ = Describe("Check loadbalancer reconcile", Serial, Ordered, func() {
 
 		})
 
+		assertFirstNode := func(nodeName string) {
+			GinkgoHelper()
+			Eventually(func() error {
+				err := k8sClient.Get(ctx, types.NamespacedName{Name: "default--node-test1", Namespace: "default"}, &lb)
+				if err != nil {
+					return err
+				}
+				if lb.Spec.Endpoints == nil || len(lb.Spec.Endpoints) != 1 {
+					return fmt.Errorf("no or more than one endpoint in LB found: %v", lb.Spec.Endpoints)
+				}
+				if len(lb.Spec.Endpoints[0].Addresses) != 1 {
+					return fmt.Errorf("no or more than one endpoint address in LB found: %v", lb.Spec.Endpoints[0].Addresses)
+				}
+				if lb.Spec.Endpoints[0].Name == nodeName &&
+					lb.Spec.Endpoints[0].Addresses[0] == "10.10.10.10" {
+					return nil
+				}
```

Comment:
Could we move the happy case to the end.

Reply:
Done.